pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -871,3 +887,25 @@ func cephVolumeEnvVar() []v1.EnvVar {
 		{Name: "CEPH_VOLUME_SKIP_RESTORECON", Value: "1"},
 	}
 }
+
+func (c *Cluster) osdRunFlagTuningOnPVC(osdID int, storageClassName string) error {
+	who := fmt.Sprintf("osd.%d", osdID)
+	do := make(map[string]string)
+
+	if storageClassName == "gp2" {
+		do["osd_recovery_sleep"] = osdRecoverySleepOnGP2
+		do["osd_snap_trim_sleep"] = osdSnapTrimSleepOnGP2
+		do["osd_delete_sleep"] = osdDeleteSleepOnGP2
```

Comment:
Can we add a comment about what these settings will do? They aren't obvious at least to me.